mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -324,6 +365,53 @@ func generateBigtableGCPolicy(d *schema.ResourceData) (bigtable.GCPolicy, error)
 	return policies[0], nil
 }
 
+func getGCPolicyFromJSON(p map[string]interface{}) (bigtable.GCPolicy, error) {
+	policies := []bigtable.GCPolicy{}
+	if p["rules"] == nil {
+		return bigtable.NoGcPolicy(), nil
```

Comment:
Is this correct? If "gc_rules" is set, but no "rules"; the input probably should be invalid.

Meaning the only case we return bigtable.NoGcPolicy() is none of the gc fields are set => !aok && !vok && !gok.

Reply:
You're right - changing the output to `nil, fmt.Errorf("...")`